There is a build error:
```
pkg/crosscluster/logical/logical_replication_job.go:93:5: r.updateStatusMessage undefined (type *logicalReplicationResumer has no field or method updateStatusMessage)
```

--- a/pkg/crosscluster/logical/logical_replication_job.go
+++ b/pkg/crosscluster/logical/logical_replication_job.go
@@ -90,6 +90,7 @@ func (r *logicalReplicationResumer) handleResumeError(
 	ctx context.Context, execCtx sql.JobExecContext, err error,
 ) error {
 	if err == nil {
+		r.updateRunningStatus(ctx, "")
 		return nil
 	}
 	if jobs.IsPermanentJobError(err) {

